### DISCUSSION TITLE
How to retrieve 'Active Power' from Shelly 1PM ?

### DISCUSSION THREAD

**Discussion by uefvanscheld** [arendst/Tasmota#16162] [Q&A] [answered]
Hi, 
today I flashed my Shelly 1PM with latest Tasmota and I'm able to obtain values like EnergyToday, EnergyTotal and more via http requests.
May be I've just overseen but haven't found a command to retrieve the current power the connected device is consuming (e.g. 42 Watts), like displayed in the WebGUI.
Can somebody help with a hint ?

Thx 
Uef

**Comment by sfromis** [comment, marked as answer]
The expected workflow is that you use the automatically published (via MQTT, **not** using http) payloads with the data, every TelePeriod (default 5 minutes), and with `PowerDelta` to create extra reporting when the energy consumption level changes.

You can also get the current sensor values using the command `Status 10`.

**Comment by uefvanscheld** [comment]
Thanks sfromis, that's what I was looking for.
And thx for the guidance how to approach this.

I normally use MQTT, but this specific target environment doesn't have a MQTT broker or other home automation center and I need to query the Shelly from another ESP32 device running a web server (beside several other tasks) to provide graphs of also other data, but including those from the Shelly.
So I want to avoid the overhead of MQTT server + client just to transfer data from one to the other device.
`Status 10 ` is at least a plan 'B'
